README.md

```diff
@@ -1310,6 +1311,42 @@ Jwts.parser()
     // ... etc ...
 ```
 
+<a name="json-jackson-custom-parsing"></a>
+### Parsing of Custom Claim Types
+
+By default JJWT will only convert simple claim types: String, Date, Long, Integer, Short and Byte.  If you need to deserialize other types you can configure the `JacksonDeserializer` by passing a `Map` of claim names to types in through a constructor. For example:
+
+```java
+new JacksonDeserializer(Maps.of("user", User.class))
+```
+
+This would trigger the value in the `user` claim to be deserialized into the custom type of `User`.  Given the claims body of:
+
+```json
+{
+    "issuer": "https://example.com/issuer",
+    "user": {
+      "firstName": "Jill",
+      "lastName": "Coder"
+    }
+}
+```
+
+The `User` object could be retrieved from the `user` claim with the following code:
+
+```java
+Jwts.parserBuilder()
+
```

Comment:
Why the blank line?  I don't think we do this in other parts of the doc, do we?  Whatever we do, we should keep it consistent.

Reply:
I was just trying to follow what I thought was the existing format, i.e.:

```java
Jwts.parser()
    
    .setAllowedClockSkewSeconds(seconds) // <----
    
    // ... etc ...
    .parseClaimsJws(jwt);
```

My personal pref would be to strip the empty lines.  My assumption was probably wrong, maybe the convention is to insert a new line between each line of code? (and keep any relevant comments directly above a given line)